lib/bolt/inventory/group2.rb

```diff
@@ -98,18 +98,27 @@ def validate_config_plugin(data, key, group_name = nil)
       def config_only_plugin(data)
         Bolt::Util.walk_vals(data) do |value|
           if value.is_a?(Hash) && value.include?('_plugin')
-            unless (plugin = @plugins.by_name(value['_plugin']))
-              raise ValidationError.new("Config lookup specifies an unknown plugin: #{value['_plugin'].inspect}", @name)
+            plugin_name = value['_plugin']
+            begin
+              hook = @plugins.get_hook(plugin_name, :inventory_config)
+            rescue Bolt::Plugin::PluginError => e
+              raise ValidationError.new(e.message, @name)
             end
-            unless plugin.hooks.include?('inventory_config')
-              raise ValidationError.new("#{plugin.name} does not support inventory_config.", @name)
+
+            begin
+              validate = @plugins.get_hook(plugin_name, :validate_inventory_config)
```

Comment:
Should this be `hook` instead of `validate` to avoid conflicting with the method of the same name?